test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -8,4 +8,23 @@ Describe "New-PSSession basic test" -Tag @("CI") {
             $_.FullyQualifiedErrorId | Should Be "InvalidOperation,Microsoft.PowerShell.Commands.NewPSSessionCommand"
         }
     }
+}
+
+Describe "JEA session bug fix test" -Tag @("Feature", 'RequireAdminOnWindows') {
```

Comment:
Please make the title more informative for the test.
Sample - "Get-Help in JEA sessions".